#! /bin/sh /usr/share/dpatch/dpatch-run

## 01_vdr_1.2.6-3.1.diff.gz.dpatch by Thomas Schmidt <tschmidt@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: New patch generated from vdr 1.2.6-3.1 diff.gz

@DPATCH@
diff -urNad vdr-1.3.23/PLUGINS/src/hello/Makefile /tmp/dpep.EDdlNh/vdr-1.3.23/PLUGINS/src/hello/Makefile
--- vdr-1.3.23/PLUGINS/src/hello/Makefile	2003-12-21 16:47:22.000000000 +0100
+++ /tmp/dpep.EDdlNh/vdr-1.3.23/PLUGINS/src/hello/Makefile	2005-04-09 21:25:03.085146680 +0200
@@ -16,7 +16,7 @@
 ### The C++ compiler and options:
 
 CXX      ?= g++
-CXXFLAGS ?= -O2 -Wall -Woverloaded-virtual
+CXXFLAGS ?= -fPIC -O2 -Wall -Woverloaded-virtual
 
 ### The directory environment:
 
diff -urNad vdr-1.3.23/PLUGINS/src/osddemo/Makefile /tmp/dpep.EDdlNh/vdr-1.3.23/PLUGINS/src/osddemo/Makefile
--- vdr-1.3.23/PLUGINS/src/osddemo/Makefile	2003-12-21 16:47:26.000000000 +0100
+++ /tmp/dpep.EDdlNh/vdr-1.3.23/PLUGINS/src/osddemo/Makefile	2005-04-09 21:25:03.085146680 +0200
@@ -16,7 +16,7 @@
 ### The C++ compiler and options:
 
 CXX      ?= g++
-CXXFLAGS ?= -O2 -Wall -Woverloaded-virtual
+CXXFLAGS ?= -fPIC -O2 -Wall -Woverloaded-virtual
 
 ### The directory environment:
 
diff -urNad vdr-1.3.23/PLUGINS/src/sky/Makefile /tmp/dpep.EDdlNh/vdr-1.3.23/PLUGINS/src/sky/Makefile
--- vdr-1.3.23/PLUGINS/src/sky/Makefile	2003-12-21 16:47:31.000000000 +0100
+++ /tmp/dpep.EDdlNh/vdr-1.3.23/PLUGINS/src/sky/Makefile	2005-04-09 21:25:03.085146680 +0200
@@ -16,7 +16,7 @@
 ### The C++ compiler and options:
 
 CXX      ?= g++
-CXXFLAGS ?= -O2 -Wall -Woverloaded-virtual
+CXXFLAGS ?= -fPIC -O2 -Wall -Woverloaded-virtual
 
 ### The directory environment:
 
diff -urNad vdr-1.3.23/PLUGINS/src/status/Makefile /tmp/dpep.EDdlNh/vdr-1.3.23/PLUGINS/src/status/Makefile
--- vdr-1.3.23/PLUGINS/src/status/Makefile	2003-12-21 16:47:41.000000000 +0100
+++ /tmp/dpep.EDdlNh/vdr-1.3.23/PLUGINS/src/status/Makefile	2005-04-09 21:25:03.085146680 +0200
@@ -16,7 +16,7 @@
 ### The C++ compiler and options:
 
 CXX      ?= g++
-CXXFLAGS ?= -O2 -Wall -Woverloaded-virtual
+CXXFLAGS ?= -fPIC -O2 -Wall -Woverloaded-virtual
 
 ### The directory environment:
 
